media/js/mozorg/contribute/contribute-2015-ga.js

```diff
@@ -154,9 +137,18 @@ $(function () {
 
         $(this).off('submit');
 
+        var selectedArea;
+        $('.area').each(function() {
+            if($(this).css('display') === 'block') {
+                selectedArea = $(this).find('select option:selected').text();
+            }
+        });
```

Comment:
I think it would be better to keep the `$('.area select').on('change')` handler to store the selected `area`, then reference that value when the form is submitted. Does that make sense? Something like:

```
$('.area select').on('change', function() {
    $('#inquiry-form').data('contribute-area', this.value);
});
```

And then:

```
window.dataLayer.push({
    event: 'contribute-signup-submit',
    contributeArea: $('#inquiry-form').data('contribute-area'),
    countrySelected: $('#id_country option:selected').text()
});
```

That way we wouldn't have to look at each `.area` or rely on CSS values.